cmd/image-builder/github-workflow-integration.md

```diff
@@ -2,7 +2,7 @@
 
 The Image Builder solution integrates with GitHub workflows and uses an Azure DevOps pipeline to run the process of building OCI
 images. It leverages a signed JWT format in which an OIDC token from GitHub's OIDC identity provider is passed. This token is used for
-secure and authorized passing of information about the workflow and the image to build to the `oci-image-builder` pipeline. The build
+authentication and authorization the reusable workflow that trigger the `oci-image-builder` pipeline. The build
```

Comment:
```suggestion
authentication and authorization in the reusable workflow that trigger the `oci-image-builder` pipeline. The build
```

Reply:
We are not using a token in the reusable workflow. We are using it in oci-image-builder pipeline to authenticate a reusable workflow. I would keep this sentence as it is.